deploy/infrastructure/modules/terraform-aws-dss/README.md

```diff
@@ -0,0 +1,74 @@
+# terraform-aws-dss
+
+This terraform module creates a Kubernetes cluster in Amazon Web Services using the Elastic Kubernetes Service (EKS) 
+and generates the tanka files to deploy a DSS instance.
+
+
+## Getting started
+
+### Prerequisites
+
+Download & install the following tools to your workstation:
+
+1. Install [terraform](https://developer.hashicorp.com/terraform/downloads).
+2. Install tools from [Prerequisites](../../../../build/README.md)
+3. Install provider specific tools:
+    1. [Amazon Web Services](./README.md#amazon-web-services)
```

Comment:
It is actually correct. Markdown creates an anchor for each titles and it can be reference using the url's fragment. In this case, it points to the next section in the same document.
This prepares the ground for an aggregated README with all cloud providers.